makefiles/Makefile.sim

```diff
@@ -29,7 +29,7 @@
 
```

Comment:
Given this file contains none of its previous content, I think we can easily replace this header with a SPDX one